### ISSUE TITLE
Lost some photos? during switch to Paleo from ES

### ISSUE THREAD

**Issue by mbprondzinski** [ArctosDB/arctos#3472]
I believe I lost some of the attached media when we switched from ALMNH:ES to ALMNH:Paleo.  I'm sure I had photos attached to a couple of records that no longer have them.   How do I retrieve them?  Thanks...

**Comment by dustymc** [comment]
I don't think these things can possibly be related, but if you'll provide specifics we can look around.

**Comment by mbprondzinski** [comment]
Okay:  ALMNH:ES:32 and ALMNH:ES:33 should have had a photo of the tusks mounted in the Chancellor's office.  At the very least, the picture would have been associated with the loan:  ALMNH:ES 2019.3
If you can find that photo, then I will give you a couple of other missing photos.  Thanks!

**Comment by dustymc** [comment]
I don't have access to those collections so I can't check the UI, but nothing in Lam's scripts was anywhere near anything involved in media links, there are no broken links between media and cataloged items or media and loans, and there's nothing about media in the changelog for those records. Some more information about the media could be useful.

**Comment by mbprondzinski** [comment]
I'll have to wait until I go into the office to find them which won't be until tomorrow.  I will keep you posted.

**Comment by mbprondzinski** [comment]
Okay, the two photos that I'm sure were uploaded because I am unable to upload them again are:  1985.6.2.jpg and Mammoth Tusks.jpg   Is that all you need?
![1985 6 2](https://user-images.githubusercontent.com/43214224/109392364-54c36b80-78e1-11eb-8630-89f9452eab92.JPG)
![Mammoth Tusks](https://user-images.githubusercontent.com/43214224/109392377-63118780-78e1-11eb-8504-ad83e323022d.jpg)

**Comment by Jegelewicz** [comment]
I'm guessing that all of your media needs new relationships to the ALMNH:Paleo catalog numbers. It should be fairly easy if you can get a list of all media with relationships to ALMNH:ES catalog numbers. I'll see if I can get that for you.

**Comment by mbprondzinski** [comment]
The Tusks jpeg was associated with the loan:  ALMNH:ES 2019.1

The stump jpeg was associated with ALMNH:ES:14512   

If you can help, that will be great!  There may be others, but these are the only two I've stumbled on so far.

BTW, I also lost a Geology record prior to the change.  I have no idea how it got deleted? but I noticed when I was setting up the Geology records for the change that one was missing.  The old number was:  ALMNH:ES:20158

**Comment by dustymc** [comment]
> unable to upload them again

The information from that error should have contained everything needed to solve this - just search Media by the MD5 checksum it provides.

> media needs new relationships to the ALMNH:Paleo catalog numbers.

That's not how this works.

>media with relationships to ALMNH:ES c

There can be none.

```
arctosprod@arctos>> select count(*) from cataloged_item where collection_id=156;
 count 
-------
     0
```

One Media file is https://arctos.database.museum/media/10604175

I don't think it's ever had any other relationships.

The %1985% are 

https://arctos.database.museum/media/10587147
https://arctos.database.museum/media/10587145
https://arctos.database.museum/media/10628205

They're attached to a record, but not one I can see.

There is only one mechanism by which a catalog record may be deleted from Arctos - https://handbook.arctosdb.org/how_to/How-to-Delete-Specimen-Records.html

**Comment by mbprondzinski** [comment]
I see it was associated with the Project, not the loan.  Sorry, I fixed it.

**Comment by Jegelewicz** [comment]
@mbprondzinski does that resolve everything?

The stump media is associated with https://arctos.database.museum/guid/ALMNH:Paleo:14510 and/or https://arctos.database.museum/guid/ALMNH:Paleo:14511

**Comment by Jegelewicz** [comment]
I see https://arctos.database.museum/guid/ALMNH:Paleo:20158 just fine, but it seems like a paleo record...

**Comment by mbprondzinski** [comment]
I copied the wrong photo number.  They all look the same!  I realized that the tusk photo was associated with the project, not the loan, so I added that relationship.  The missing record I would swear was originally consecutively numbered with the other Geology specimens, which is why I questioned it.  It was a piece of petrified wood.  I saw that the number was assigned to a PV specimen which seems odd because it is right in the middle of the geology specimens which are all linked to one repeated loan!  (They are the only Geo specimens that I had assigned catalog numbers to.)  Anyway, I will have to assign a new catalog number to that missing specimen regardless.

I suspect the other "missing" photo was never uploaded.  I have many duplicates, so it becomes a tangled web, especially since some of them had no signage attached to them.

Thanks for everyone's help on this!  Nothing like a Monday to get the week rolling !
